clearwater-cluster-manager.root/usr/share/clearwater/clearwater-cluster-manager/scripts/load_from_cassandra_cluster.py

```diff
@@ -27,8 +27,22 @@
     # output as valid yaml, we need to use tr to replace tabs with spaces.
     # We remove any xss=.., as this can be printed out by 
     # cassandra-env.sh
-    command = "/usr/share/clearwater/bin/run-in-signaling-namespace nodetool describecluster | grep -v \"^xss = \" | tr \"\t\" \" \""
-    desc_cluster_output = subprocess.check_output(command, shell=True)
+    nodetool_cmd = ['/usr/share/clearwater/bin/run-in-signaling-namespace',
+            'nodetool', 'describecluster']
+    grep_cmd = ['grep', '-v', '\"^xss = \"']
+    trace_cmd = ['tr',  '\"\t\" \" \""']
```

Comment:
I think some of these escapes are unnecessary (remember, these are now not being invoked by a shell so one round of escaping has been eliminated).